keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -350,26 +350,26 @@ At a high level, this proposal is based around three new concepts:
 
 The implementation of these concepts requires several changes to the
 ResourceSlice API. It introduces two new fields on the `ResourceSliceSpec`.
-`CapacityPools` defines a list of `CapacityPools` while `Mixins` define the
+`SharedCounters` defines a list of `CounterSet` while `Mixins` define the
 mixins. It also introduces a new field `PerDeviceNodeSelection` on the
 `ResourceSliceSpec` and new fields on the device that mirrors the node selector
 fields on the `ResourceSlice`.
 
-1. The `CapacityPools` field is a list of named `CapacityPool`s. Each
+1. The `SharedCounters` field is a list of named `CounterSet`s. Each
    defines a set of capacities that is available for devices. This makes it possible
```

Comment:
s/capacities/counters/

There are many places this is needed below, along with `s/capacity pool/counter set/` etc.